Puzzlepart/prosjektportalen365
# Update ReactMarkdown to latest throughout solution

### Your checklist for this pull request


### Description

Information messages now uses a newer and stable version to render HTML, see: #762 for more description.

### How to test

1. Open TemplateSelector dialog to verify that the text in the information dialog is rendered properly and the link is clickable

### Relevant issues (if applicable)

#762
